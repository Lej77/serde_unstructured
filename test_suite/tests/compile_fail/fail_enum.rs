
#[derive(serde_unstructured::SerdeView)]
enum Data {
    Variant,
}

fn main() {}