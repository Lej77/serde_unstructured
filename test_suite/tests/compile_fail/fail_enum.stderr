error: The `SerdeView` derive macro can only be applied to structs.
 --> tests/compile_fail/fail_enum.rs:2:10
  |
2 | #[derive(serde_unstructured::SerdeView)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `serde_unstructured::SerdeView` (in Nightly builds, run with -Z macro-backtrace for more info)
